### PR TITLE
feat(SEO): add data

### PR DESCRIPTION
**Pull request by andreybakanovsky**

### CODE REVIEW

**Comment by andreybakanovsky** [comment]
@ka8725 could you offer recommendations on how I could improve the meta tags? Please, just set the direction

**Review by VladislavSokov — APPROVED** [review]

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,6 +6,29 @@
...
+    <meta property="og:image" itemprop="image" content=<% image_tag 'budgeting_kid_landing_0.jpg'%> >
+
+    <meta property="twitter:card" content='summary_large_image'>
+    <meta property="twitter:url" content="http://budgetingkid.com/">
+    <meta property="twitter:title" content="BudgetingKid - Foster good financial habits in your children">
+    <meta property="twitter:description" content="Empower Your Kids with Financial Wisdom: Cultivate responsible money habits
+      from an early age. Our application makes it effortless to guide your children towards a secure financial future.
+      Teach them to manage, save, and plan wisely with ease. Track allowances, set goals, and collaborate seamlessly
+      with family members. Start their journey to financial literacy today!">
+    <meta property="twitter:image" content=<%image_tag 'budgeting_kid_landing_0.jpg'%>>
```
Comment (ka8725) [inline app/views/layouts/application.html.erb:31]: Should not the value of content property be surrounded with ""?

**Comment by ka8725** [comment]
> @ka8725 could you offer recommendations on how I could improve the meta tags? Please, just set the direction

In which direction you are looking for improvement? Technically they look ok. I have just one note

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,6 +6,29 @@
...
+      with family members. Start their journey to financial literacy today!">
+
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="http://budgetingkid.com/">
+    <meta property="og:title" content="BudgetingKid - Foster good financial habits in your children">
+    <meta property="og:description" content="Empower Your Kids with Financial Wisdom: Cultivate responsible money habits
+      from an early age. Our application makes it effortless to guide your children towards a secure financial future.
+      Teach them to manage, save, and plan wisely with ease. Track allowances, set goals, and collaborate seamlessly
+      with family members. Start their journey to financial literacy today!">
+    <meta property="og:image" itemprop="image" content=<% image_tag 'budgeting_kid_landing_0.jpg'%> >
```
Comment (ka8725) [inline app/views/layouts/application.html.erb:22]:
```suggestion
    <meta property="og:image" itemprop="image" content="<% image_tag 'budgeting_kid_landing_0.jpg'%>" >
```

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ka8725) [inline app/views/layouts/application.html.erb:31]:
```suggestion
    <meta property="twitter:image" content="<%image_tag 'budgeting_kid_landing_0.jpg'%>">
```

**Pull request merged**